distribution/docs/src/main/resources/content/_tables/Session.adoc

```diff
@@ -19,7 +19,7 @@
 |Session Timeout (in minutes)
 |expirationTime
 |Integer
-|The number of minutes after a session has been inactive that it should be invalidated.
+|Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 31 minutes.
```

Comment:
I thought the default timeout was 15 minutes. Or is that something else? @peterhuffer ?

Reply:
The 15 min default is for `ddf.platform.ui.config`.